src/redux/core/voting/thunks.js

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
What goal of creating this file?
- unused createPollAndNavigateToWallets
- removePollAndNavigateToVoting used only in RemovePollTopButton. Also, it better to dispatch createPoll and navigation actions in appropriate handler in component.
- unused createPollAndNavigateToVoting
actionAndAction names looks like antipattern
If action has not certain navigation point, navigation actions should be dispatched from component. IMO, all navigation actions should be dispatched from components or containers.